packages/react-mutation-mapper/src/store/DefaultMutationMapperFilterApplier.ts

```diff
@@ -20,6 +19,7 @@ import {
     applyDefaultProteinImpactTypeFilter,
 } from '../util/FilterUtils';
 import { defaultOncoKbFilter } from '../util/OncoKbUtils';
+import { IOncoKbData } from 'cbioportal-frontend-commons';
```

Comment:
move this all the way up before internal imports? or at least group with other models.